backend/corpora/common/corpora_orm.py

```diff
@@ -247,13 +247,12 @@ class DbDataset(Base, AuditMixin):
     tombstone = Column(Boolean, default=False, nullable=False)
     original_id = Column(String)
     published = Column(Boolean, default=False)
+    explorer_s3_uri = Column(String)
```

Comment:
I'm curious -- in the upload processing step, we create a row for each artifact created (i.e. H5AD, RDS, and LOOM), but not CXG. And yet we have a CXG `DatasetArtifactFileType` type. Why was that decision made and why link the artifact s3_uri here instead of populating another row in the DatasetArtifact table with the CXG type? Linking it here seems to violate the semantics of that table's purpose.

Reply:
We do have some CXG artifacts in the database and in the data-portal s3. We use it early on but stopped. I don't see why we couldn't do that. It would require modifying the database to include a URL field. It would also require us to modify the backend or frontend code to know where to find the CXG URL.

Reply:
I think the natural step was to go from a list of `DbDeploymentDirectory` which generally only ever contained one entry and had no fields in common with `DbDatasetArtifact`, back to the DATASET table. Now that we added `explorer_s3_uri` we now have one field that overlaps with what in the `DbDatasetArtifact`. I'm on the fence about making it a `DbDatasetArtifact`, but it wouldn't be too much work to make it happen.

Reply:
I think it makes sense to make it a DbDatasetArtifact instead of adding the explorer_s3_uri. I can update this pr to remove the field and create a ticket to migrate the data/create an artifact to represent the s3_uri